### DISCUSSION TITLE
Add Capital Letters function in used Wysiwyg

### DISCUSSION THREAD

**Discussion by JustinKoenig123** [directus/directus#11932] [Feature Requests]
Hey there, we are working with your added wysiwyg editor. A Capital Letter function within the toolbar of the editor would be nice.

**Comment by rijkvanzanten** [comment]
Capital letter function? Do you mean like a drop cap?

**Reply by JustinKoenig123** [reply]
Sorry, I wrote stupidly. I meant that a selected text is converted to uppercase using a button press. Behavior like the bold function in the Wysiwyg toolbar.

**Reply by rijkvanzanten** [reply]
Ahh gotcha. Are you thinking to Capitalize, or make UPPERCASE? Or Maybe Even do it Title Formatted?

**Reply by JustinKoenig123** [reply]
It is actually sufficient to select the text and display it in capital letters by pressing a button in the taskbar. Or you press the button and all following characters are written in capital letters. Similar to Bold.

**Reply by rijkvanzanten** [reply]
> Or you press the button and all following characters are written in capital letters. Similar to Bold.

I believe this is what CAPS LOCK is for 😄

**Reply by JustinKoenig123** [reply]
coming back after over a year and i'm still laughing about that feature request :laughing: :rofl:

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
